### ISSUE TITLE
User feedback that we are not capturing all his data

### ISSUE THREAD

**Issue by osilla** [sourcerer-io/sourcerer-app#190]
Cool initiative, but something is broken it lists way too little data. https://sourcerer.io/rogeralsing Go: 881 loc. listed in the report only.

see attached screen shot

![screenshot_20180203-124401](https://user-images.githubusercontent.com/20307788/35769905-440ae42c-08c7-11e8-9388-5e57835b7b54.jpg)

**Comment by mcg1969** [comment]
Indeed, very little useful data is going to be found for me because mine is all tied up in the commits I make to organization-owned projects. And supposedly, I gave sourcerer permission to access those repos, so it's not clear why it doesn't see it.

**Comment by BlitzKraft** [comment]
I am facing a somewhat similar issue - only one repo of mine is being processed. 103 repos, 5 OKs, one fail and the rest are skipped. Among the processed ones, there is only one repo for the stats. Here's my profile: 

https://sourcerer.io/blitzkraft

I am not sure if I should open a new issue for this.

**Comment by sergey-surkov** [comment]
@BlitzKraft it looks like for many commits GitHub does not know your email addresses. They are attributed to "BK Bolisetty". Sourcerer relies on email addresses for commit attribution. If you don't want to make your email public, could you please send us any message from it to hello@sourcerer.io? This will work as a confirmation for you, and we'll associate the email address with your account manually. After that, the attribution should work fine.

**Comment by sergey-surkov** [comment]
@rogeralsing : your profile seems pretty complete now. you may want to re-do sourcerer.io/start to update it. you can also use your pic from github now. please let us know if there still an issue.

**Comment by sergey-surkov** [comment]
@anatolykislov could you please look into the issue @mcg1969 is seeing?

**Comment by rogeralsing** [comment]
Some questions, why is it listing 2015-2016 only?
<img width="645" alt="skarmavbild 2018-02-23 kl 08 00 19" src="https://user-images.githubusercontent.com/647031/36581957-b42866dc-186f-11e8-8771-1ecdc3428894.png">

I started doing Kotlin last year, and that is on the report.
I've been doing Go for multiple years, and that is mostly missing (it says 800loc, which is incorrect)

<img width="419" alt="skarmavbild 2018-02-23 kl 08 02 19" src="https://user-images.githubusercontent.com/647031/36582006-e8d6e1ce-186f-11e8-99a8-c1952f09d55b.png">
(from my org-repo https://github.com/AsynkronIT/protoactor-go)

Or is it only including direct personal repos?

**Comment by mcg1969** [comment]
I redid the start page and it picked up a lot more. But oddly, it skipped 2 repos because I had more than 300 commits in each. (The full text of the stated reason was truncated.) That seems quite odd and of course counterproductive if the purpose is to understand our coding behavior.

**Comment by mcg1969** [comment]
Ah, so the full message is `+300 commits. Use App`. FYI. It really doesn't make sense to me to exclude my two most active repos from display on my public profile.

**Comment by redbitshift** [comment]
guys we are looking into it. will keep you updated.

**Comment by sergey-surkov** [comment]
mcg1969, the commit limit is now raised to 10,000.

**Comment by rogeralsing** [comment]
Also, as I don't use Python, "I like list comprehensions in python" this feels out of place tbh :)

**Comment by mcg1969** [comment]
It is getting better! Alas it still does not see my most "Important" org https://github.com/cvxr, and it confuses MATLAB for objective-c.

**Comment by mcg1969** [comment]
Ah, restarting picked up the CVXR repos.

**Comment by sergey-surkov** [comment]
@rogeralsing acknowledging the bug. looking into it.
